acs-integration-tests/testng-predix-cloud.xml

```diff
@@ -40,8 +35,14 @@
             <class name="com.ge.predix.integration.test.PolicyEvaluationWithAttributeConnectorIT"/>
             <class name="com.ge.predix.integration.test.PredixAuditIT"/>
             <class name="com.ge.predix.integration.test.PrivilegeManagementAccessControlServiceIT"/>
-
             <class name="com.ge.predix.acs.cloudfoundry.DeleteApplications"/>
         </classes>
     </test>
-</suite>
+    <test name="predix-cloud-cucumber">
```

Comment:
Maybe a better name for the test is predix-cloud-sequential?
Same comments applies to all suites.